test/geometry/adjoints.jl

```diff
@@ -53,25 +53,56 @@ end
     end # TODO: Remove the @test_broken after tests work.
 end
 
-@testset "SVector{N, T} gradient" begin
+@testset "SVector{N, T} and SMatrix{N, N, T, L} gradient" begin
     for t in (Float32, Float64, RGB{Float32}, RGB{Float64})
         inp = rand(t, 2)
+        inp_mat = rand(t, 3, 3)
         _sep(x) = x.r + x.g + x.b
         if t ∈ (Float32, Float64)
             @test Zygote.gradient(x->sum(SVector{2, t}(x)), inp)[1] == ones(t, 2)
+            @test Zygote.gradient(x->sum(SMatrix{3, 3, t, 9}(x)), inp_mat)[1] == ones(t, 3, 3)
         else
             @test Zygote.gradient(x->_sep(sum(SVector{2, t}(x))), inp)[1] == ones(t, 2)
+            @test Zygote.gradient(x->_sep(sum(SMatrix{3, 3, t, 9}(x))), inp_mat)[1] == ones(t, 3, 3)
         end
     end
 end
 
 @testset "DiffImages.Homography method gradient" begin
-    h = DiffImages.Homography()
+    h = DiffImages.Homography{Float64}()
     v = rand(SVector{2, Float64})
     zy = Zygote.gradient((x,y)->sum(y(x)), v, h)
     fd = grad(central_fdm(5,1), (x,y)->sum(y(x)), v, h)
-    @test zy[1] ≈ fd[1]
```

Comment:
Why are we getting rid of the test?

Reply:
It got removed accidentally. Thanks for reminding me, I have added a better version of the same now.